### ISSUE TITLE
Permissions

### ISSUE THREAD

**Issue by davidrpugh** [ScalABM/docker-stacks#2]
@ibillett 

I have re-instated write permissions for this repo (and `docker-base`)  to the `docker` team.  You are the maintainer for this team (which I thought should give you write permission but apparently not).  I can not seem to work out how to give only one person on a team write permissions to the repos controlled by that team.

**Comment by ghost** [comment]
Based on what you have done, I now have the ability to create new branches in the repo, but not commit directly to `master` because it is ['protected'](https://help.github.com/articles/about-protected-branches/). 

Must be the case that only the creator can commit / merge PRs into master? 

I'll do my best to keep the PRs short and sweet 😎

**Comment by davidrpugh** [comment]
@ibillett protecting the master branch is a "best practice" it prevents, among other things, collaborators accidentally deleting the branch.  Create a new feature branch with your changes and push the branch to this repo and then submit a PR.

**Comment by davidrpugh** [comment]
@ibillett Has your permissions issue been sorted? Let me know.  We seemed to have lost momentum and I would like get it back :)

**Comment by ghost** [comment]
@davidrpugh Yes I think so, managed to push changes to the [example-auctions branch](https://github.com/EconomicSL/docker-stacks/tree/example-auction/base-python). I need to sit down and figure out security permissions before submitting a PR.

Apologies for the lack of comms, it has been a crazy busy couple of weeks at work.
